### DISCUSSION TITLE
The add-emoji icon is visible in every post's reaction footer when there are no reactions

### DISCUSSION THREAD

**Discussion by jay** [community/community#50852] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

GitHub recently changed it so that issue and pull request threads show the add emoji icon for each post regardless of whether there are any reactions. Prior to this change the emoji icon only showed at the top of each post, until there was a reaction and then it would show next to the emojis as well.

Old:

![old](https://user-images.githubusercontent.com/965580/227326624-7de20c44-1e3c-41df-918f-df13fcfd2414.png)

New:

![new](https://user-images.githubusercontent.com/965580/227326694-17281fda-e109-4696-9359-510a188c5d09.PNG)

In my opinion the old way is better. As I scan through threads now it looks to me like every comment has an emoji even though it doesn't, it's just the add emoji button by itself.

When I use another browser with a clean profile and not logged in then I don't see this behavior. Also, when I am logged in I see [emojis showing as gray font characters](https://github.com/orgs/community/discussions/47821) and not actual graphics.

**Comment by philBrown** [comment]
Total waste of vertical space too on pages where vertical space is at a premium

**Comment by tarydon** [comment]
Please roll this back - it's extremely annoying to see a bunch of vapid "smiling face" emojis when trying to get serious work done.
